framework/resources/Functional/filter/pushdown/filter_pruning/DRILL_6865_no_filter_pruning_expression_in_filter.plan

```diff
@@ -0,0 +1,2 @@
+Filter
```

Comment:
Are we sure to keep this as just "Filter" without regex? Like '.*'

Also here and in other tests, we should test the filter condition as well - like '< 5', plus columns being pushed down etc,

Reply:
We don't need to match a whole line with an operator, so wildcard is used only inside the matching strings.
Added conditions and columns except one case, on which a new JIRA was submitted: DRILL-6910.

Reply:
Thanks for sharing!